examples/elasticsearch/src/buildRequest.js

```diff
@@ -0,0 +1,121 @@
+import buildRequestFilter from "./buildRequestFilter";
+
+function buildFrom(current, resultsPerPage) {
+  if (!current || !resultsPerPage) return;
+  return (current - 1) * resultsPerPage;
+}
+
+function buildSort(sortDirection, sortField) {
+  if (sortDirection && sortField) {
+    return [{ [`${sortField}.keyword`]: sortDirection }];
+  }
+}
+
+function buildMatch(searchTerm) {
+  return searchTerm
+    ? {
+        multi_match: {
+          query: searchTerm,
+          fields: ["title", "description"]
+        }
+      }
+    : { match_all: {} };
+}
+
+/*
+
+  Converts Current Application State to an Elasticsearch request.
```

Comment:
No need to case Current Application State.
```suggestion
  Converts current application state to an Elasticsearch request.
```